Core/Jira/JiraVersionCreator.cs

```diff
@@ -42,7 +42,7 @@ public string CreateNewVersionWithVersionNumber (string versionNumber)
   {
     var jiraProject = _config.Jira.JiraProjectKey;
     if (string.IsNullOrEmpty(jiraProject))
-      throw new InvalidOperationException("Jira project was not assigned.");
+      throw new InvalidOperationException("There is no JIRA project key specified in the config. Stopping execution");
```

Comment:
```suggestion
      throw new UserInteractionException("There is no JIRA project key specified in the config. Stopping execution.");
```
(Given GitBranchOperatiosn, we could/should make this a UIEx.)